kibble.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "core-template",
   "version": "0.0.1",
-  "siteUrl": "https://abccinemas.screenplus.co",
+  "siteUrl": "https://staging-test.shift72.com",
```

Comment:
Are we moving to use this site rather than abccinemas

Reply:
I left that in by mistake. I'll fix it now.